src/ZenohUTransport.cpp

```diff
@@ -9,4 +9,384 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
+#include <up-cpp/datamodel/serializer/UUri.h>
+#include <up-cpp/datamodel/serializer/Uuid.h>
+#include <up-cpp/datamodel/validator/UMessage.h>
+#include <up-cpp/datamodel/validator/UUri.h>
+
+#include <stdexcept>
+
 #include "up-transport-zenoh-cpp/ZenohUTransport.h"
+
+namespace uprotocol::transport {
+
+const char UATTRIBUTE_VERSION = 1;
+
+const uint32_t WILDCARD_ENTITY_ID = 0x0000FFFF;
+const uint32_t WILDCARD_ENTITY_VERSION = 0x000000FF;
+const uint32_t WILDCARD_RESOURCE_ID = 0x0000FFFF;
+
+using namespace zenoh;
+using namespace uprotocol::v1;
+using namespace uprotocol::datamodel;
+
+UStatus ZenohUTransport::uError(UCode code, std::string_view message) {
+	UStatus status;
+	status.set_code(code);
+	status.set_message(std::string(message));
+	return status;
+}
+
+std::string ZenohUTransport::toZenohKeyString(
+    const std::string& default_authority_name, const UUri& source,
+    const std::optional<v1::UUri>& sink) {
```

Comment:
Confirm source / sink optionality are correct, and add some documentation so it is clear that these names are from the perspective of the sender.